.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
this file isn't necessary

Reply:
Two quick questions:
1) There isn't anything in the directories from .cache on: Is there any reason not to remove everything including .cache? (or is it better to just delete lastfailed?)
2) I see the comment about lastfailed not being necessary: is that the only comment, or is there somewhere else I should be looking for other comments?

Thanks again, @mgxd!
Kesshi

Reply:
@kesshijordan - `lastfailed` is the only one that's included in the commit. probably got added by mistake. but in general nothing in `.cache` should be in the commit.

Reply:
Thanks, @satra. I removed the .cache file from git and have pushed the change. @mgxd is there anything else I need to address?